packaging/observability/microshift-observability.service

```diff
@@ -0,0 +1,20 @@
+[Unit]
+Description=MicroShift Observability
+After=microshift.service
```

Comment:
Should we use `ConditionPathExists` here for all the files the service expects to have before it starts?

Reply:
Right, but the point of the condition in systemd is not to attempt starting the service if the path does no exist.

Reply:
Could this help to avoid unnecessary restarts?